packages/components/skeleton/src/Skeleton.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * @license
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://github.com/IDuxFE/idux/blob/main/LICENSE
+ */
+
+import { computed, defineComponent, normalizeClass } from 'vue'
+
+import { isNumber } from 'lodash-es'
+
+import { useGlobalConfig } from '@idux/components/config'
+
+import { skeletonProps } from './types'
+
+export default defineComponent({
+  name: 'IxSkeleton',
```

Comment:
```suggestion
  name: 'IxSkeleton',
  inheritAttrs: false,
```
为了避免 attrs 被透传到默认插槽，所以这里要设置一下

Reply:
已加上